lib/core/auth.js

```diff
@@ -275,7 +299,7 @@ export default class Auth {
     return this.ctx.app.$axios
       .request(_endpoint)
       .then(response => {
-        const result = _endpoint.propertyName ? getProp(response.data, _endpoint.propertyName) : response.data
+        const result = response.data
```

Comment:
We are in a breaking change anyway. What do you think to always return an object like `{ data, reponse }`?

Reply:
Yes! Simpler to use!

Reply:
Refactored in commit https://github.com/nuxt-community/auth-module/pull/548/commits/80566bf248802fc94bb9635053d98debbf05ac6c